test/resources/layout-shared.css

```diff
@@ -10,3 +10,28 @@ body {
     background: beige;
     font-size: 14px;
 }
+
+.fd-action-bar {
+    margin: 0 20px;
+}
+
+.test-btn-placeholder {
+    border: 1px solid lightblue;
+    border-radius: 4px;
+    padding: 2px 5px;
+    background-color: yellow;
+    color: green;
+}
+
+.test-btn-back {
+    margin: 0;
+    padding: 0;
+    border: none;
+    border-radius: 4px;
+    background-color: white;
+    min-height: 28px;
+    width: 26px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
Can we replace both `.test-btn-placeholder` and `.test-btn-back` with `.btn-playground`?